sdk/network/mgmt/src/main/java/com/azure/management/network/implementation/NetworkInterfaceImpl.java

```diff
@@ -69,7 +69,7 @@
 
     @Override
     protected Mono<NetworkInterfaceInner> getInnerAsync() {
-        // FIXME: 3rd parameter
+        // TODO(not known): 3rd parameter
```

Comment:
This TODO can be solved now

Reply:
Should all expand TODO remain `null`?

Reply:
If the call is using `null` it could now be deleted.

This is yaohai adding these `null`s when method call with defaults not ready.

Reply:
done